ingestion/functions/parsing/common/python/parsing_lib/parsing_lib.py

```diff
@@ -74,7 +74,7 @@ def prepare_cases(cases, upload_id):
     TODO: Migrate source_id/source_url to this method.
     """
     for case in cases:
-        case["caseReference"]["uploadId"] = upload_id
+        case["caseReference"]["uploadIds"] = [upload_id]
```

Comment:
the argument to this method should be `upload_ids` and be the array of existing uploads no?

Reply:
How I've configured it is that the middleware will take any upload_ids from the request and add it to the existing upload_ids (removing duplicates). That way a user could specify all existing upload ids in the request, or just the one new upload id that needs to be added to the existing list. This means the requester does not have to look up existing upload_ids in order to add their new uploadId to the list. 

This is an example of just supplying the new upload_id.

Reply:
oh so even if you just add a new one it has to be a list, I guess that's what tripped me. thanks for the clarification.